src/reactive-dev-formatter.ts

```diff
@@ -19,15 +19,11 @@ interface DevToolFormatter {
     body: (object: any, config: any) => any;
 }
 
-function getTarget(item: any) {
-    return item && item[TargetSlot];
-}
-
 function extract(objectOrArray: any): any {
     if (isArray(objectOrArray)) {
         return objectOrArray.map((item) => {
-            const original = getTarget(item);
-            if (original) {
+            const original = unwrap(item);
+            if (original !== item) {
```

Comment:
Is this to handle a proxy within a proxy?

Reply:
Yes. As per my understanding, getTarget(item) is a check to see if item is proxy. If it is, then it recursively extracts that proxy. The new logic is "Can this value be unwrapped? If yes, then its a proxy, recursively extract."